citi-onboarding/Maracatec
# Feature/nodemailer

### What I Did
Create nodemailer
### How to Test

1. Open the terminal and go to the folder of this project

2. In terminal:

   i. `git checkout develop`
  ii. `git pull`
 iii. `git checkout feature/nodemailer`
 iv. `git pull origin feature/nodemailer`
  v. `open the folder server with comand cd client/`
 vi. `yarn install or npm install`
vii. `yarn start or npm start`

 3. Check if the component looks just like the mockup in different screen sizes

 4.  Check if you can send an email successfully

5. Read the changed files section on GitHub

6. Test the things you think are worth testing, even the ones that are not in this description
